### ISSUE TITLE
Setting additional options does not work

### ISSUE THREAD

**Issue by zostay** [IdeaGinkgo/Intellij-Ginkgo#45]
I would like to only run a subset of packages in my test. However, if I try to set those by adding options or add any command-line at all, it appears that they get sent to ginkgo as a single string.

For example, setting the following in the test configuration's **Ginkgo additional options**:

```--fail-fast --randomize-all pkg/ install/ test/bugs/```

and then running results in the following error message followed by the ginkgo usage message:

```
/Users/sterling/go/bin/ginkgo -v "--fail-fast --randomize-all pkg/ install/ test/bugs/" -r
ginkgo run failed
  flag provided but not defined: -fail-fast --randomize-all pkg/ install/ test/bugs/
```

Those extra options just need to be processed as command-line options and split up.

**Comment by zostay** [comment]
I can identify the code that has the bug, but I don't know enough about the IntelliJ plugin SDK to know how best fix it. But basically, the code here: 

https://github.com/IdeaGinkgo/Intellij-Ginkgo/blob/3929387fd86b5e526463156b74e15f33c3a0871d/src/main/java/com/github/idea/ginkgo/execution/testing/GinkgoRunningState.java#L190,L225

Needs to be modified so that:

```
commandList.add(runOptions.getGinkgoAdditionalOptions());
```

becomes something more like:

```
for (String option : splitOptions(runOptions.getGinkgoAdditionalOptions())) {
    commandList.add(option)
}
```

The current solution only works if there's a single option in that configuration field.

**Comment by TaylorOno** [comment]
Thank you for doing the leg work I should be able to release a fix for this shortly

**Comment by TaylorOno** [comment]
This should be fixed in 0.5

**Comment by zostay** [comment]
Confirmed, though, there's a new bug with it.
